### ISSUE TITLE
FR - MT Card Row Block (2 Cards Variant)

### ISSUE THREAD

**Issue by shirin27** [hlxsites/vg-macktrucks-com#621]
From vg-macktrucks-com-rd created by [DanielaPedrochevd](https://github.com/DanielaPedrochevd): Netcentric/vg-macktrucks-com-rd#188  

**Agile Requirement**
As a business owner, I want to display content so that it follows the design, to achieve a more dynamic site: 

- [2 Cards](https://www.figma.com/design/oNwe2jHIICPMIe551EY62l/Mack-Reskin---Full-Project?node-id=2243-101562&m=dev)

**Description**

Create a new Card Row block variant (2 cards) that follows the above-mentioned design.  2 Cards is very similar to the Cards block but we would need to add the bold font as well as changing the "Explore" button to a slightly bigger one. 

**Acceptance Criteria**

1. As an editor, I can select the new (2 cards) variant from the Cards row block.
2. The new variant follows the figma design and style mentioned above.
3. As an editor, I can configure the new variant by adding an image, a title, and some text which I can select of it is displayed in bold or not, followed by a button. 

**Figma/Attachment:**

Desktop: https://www.figma.com/design/oNwe2jHIICPMIe551EY62l/Mack-Reskin---Full-Project?node-id=2243-101562&m=dev

<img width="251" alt="image" src="https://github.com/user-attachments/assets/08fb1720-398f-4574-a809-179e8394dfd0">

Mobile: https://www.figma.com/design/oNwe2jHIICPMIe551EY62l/Mack-Reskin---Full-Project?node-id=2243-102265&m=dev
<img width="42" alt="image" src="https://github.com/user-attachments/assets/ece0c657-246f-4e3a-8a90-10561ad9eabe">

**Comment by shirin27** [comment]
This ticket has to be done after #112

**Comment by AsmitKMishra** [comment]
Testing Completed. Specs match with requirements and dimensions and design match with that in Figma within the margins.
